src/test/resources/epub3/files/content-document-xhtml/aria-roles-li-valid.xhtml

```diff
@@ -13,9 +13,9 @@
 			</ol>
 		</section>
 		<section>
-			<h2>Notes</h2>
+			<h2>Bibliography</h2>
 			<ul>
-				<li role="doc-endnote">…</li>
+				<li role="doc-biblioentry">…</li>
```

Comment:
Shouldn't this appear within the (required) context role `doc-bibliography`?
Also, why remove the `doc-endnote` test?

Reply:
That's just some weirdness in the diff because I started with an existing file that had doc-endnote in it. The doc-endnote role is in the next section; it hasn't been removed.

But in response to this and your other comment, I questioned even having these two roles. They're the only required children roles and they're also overwhelmingly likely to be deprecated in the now soon-to-start 1.1 revision. Once that happens, their section roles will exist and not require them and these two will need to be moved to invalid tests.

Should we just strip these list item tests for now and put them formally into the invalid set once the revision gets going?

Reply:
> That's just some weirdness in the diff because I started with an existing file that had doc-endnote in it. The doc-endnote role is in the next section; it hasn't been removed.

Oh, sorry I missed that 👍



> Should we just strip these list item tests for now and put them formally into the invalid set once the revision gets going?

Yeah, I don't know; it also depends on how soon this will happen.
I think I'm more  in favor of updating the test files to represent the spec as it currently is, and then fix it back later…

Reply:
> it also depends on how soon this will happen

Any day now is my understanding. The ARIA group just has to form a subgroup and we'll be underway.

I can add roles onto the parent sections to make them theoretically valid, but I'd then leave the section tests alone so we cover both scenarios. That way we can drop this whole file into the invalid tests when the time comes and nothing else needs to change.

Reply:
OK, SGTM. Don't bother with making the changes, I can do it.